### ISSUE TITLE
`1.4.15` release broken due to `albucore` changes

### ISSUE THREAD

**Issue by hlky** [albumentations-team/albumentations#1945]
## Describe the bug

`1.4.15` release specifies [`albucore>=0.0.16`](https://github.com/albumentations-team/albumentations/blob/6a8d9456a52eafd050d245479b58bba583d4da83/setup.py#L13) and there are breaking changes in `albucore==0.0.17`.

### To Reproduce

1. `albumentations==1.4.15`
2. `import albumentations as A`
3. 
```
----> 1 import albumentations as A

File albumentations/__init__.py:6
      3 from albumentations.check_version import check_for_updates
      5 from ._version import __version__  # noqa: F401
----> 6 from .augmentations import *
      7 from .core.composition import *
      8 from .core.serialization import *

File albumentations/augmentations/__init__.py:1
----> 1 from .blur.functional import *
      2 from .blur.transforms import *
      3 from .crops.functional import *

File albumentations/augmentations/blur/__init__.py:1
----> 1 from .functional import *
      2 from .transforms import *

File albumentations/augmentations/blur/functional.py:9
      7 import cv2
      8 import numpy as np
----> 9 from albucore.utils import clipped, maybe_process_in_chunks, preserve_channel_dim
     11 from albumentations.augmentations.functional import convolve
     12 from albumentations.augmentations.geometric.functional import scale

ImportError: cannot import name 'preserve_channel_dim' from 'albucore.utils' (albucore/utils.py)
```

**Comment by Arthur151** [comment]
Same here. 
After 
pip install albumentations -U
forcing to downgrade the albucore, solves my bug for now:
pip install albucore==0.0.16
Waiting for the right solution.

**Comment by Prbhhh** [comment]
How to downgrade the albucore?
pip install albucore==0.0.16

**Comment by ternaus** [comment]
Released version 1.4.16 with a strict binding with a particular albucore version.

`pip install -U albumentions` should work now
